<ui version="4.0" >
 <author>Dag Andersen &lt;danders@get2net.dk></author>
 <class>KPlato::AccountsPanelBase</class>
 <widget class="QWidget" name="KPlato::AccountsPanelBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>350</width>
    <height>220</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>AccountsPanelBase</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Default account:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="accountsComboBox" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="accountList" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="newBtn" >
       <property name="text" >
        <string>&amp;New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="subBtn" >
       <property name="text" >
        <string>New &amp;Sub-Account</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBtn" >
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <includes>
  <include location="local" >k3listview.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
